editor/editor_node.cpp

```diff
@@ -491,7 +491,10 @@ void EditorNode::_notification(int p_what) {
 				}
 
 				for (int i = 0; i < addons.size(); i++) {
-					set_addon_plugin_enabled(addons[i], true);
+					if (addons[i].begins_with("res://"))
+						set_addon_plugin_enabled(addons[i], true);
+					else
+						set_addon_plugin_enabled("res://addons/" + addons[i] + "/plugin.cfg", true);
```

Comment:
Braces aren't enforced in the 3.2 branch, but I think it would be best if new additions used them:

```suggestion
					if (addons[i].begins_with("res://")) {
						set_addon_plugin_enabled(addons[i], true);
					} else {
						set_addon_plugin_enabled("res://addons/" + addons[i] + "/plugin.cfg", true);
					}
```

Reply:
Agree!